### ISSUE TITLE
Unable to access the jenkins.jx.xxxxx.nip.io/me /configure to get the API token of jenkins installed in gke cluster

### ISSUE THREAD

**Issue by Nagamohan1** [jenkins-x/jx#2352]
### Summary 
I try to create the gke cluster, getting following error 
**command output:**

```
.....................
.....................
Adding values file C:\Users\Nagamohan_Koganti\.jx\cloud-environments\env-gke\myvalues.yaml
Adding values file C:\Users\Nagamohan_Koganti\.jx\gitSecrets.yaml
Adding values file C:\Users\Nagamohan_Koganti\.jx\adminSecrets.yaml
Adding values file C:\Users\Nagamohan_Koganti\.jx\extraValues.yaml
Adding values file C:\Users\Nagamohan_Koganti\.jx\cloud-environments\env-gke\secrets.yaml
waiting for install to be ready, if this is the first time then it will take a while to download images
Jenkins X deployments ready in namespace jx


        ********************************************************

             NOTE: Your admin password is: curtainfrost

        ********************************************************

Getting Jenkins API Token
Using url http://jenkins.jx.35.234.62.96.nip.io/me/configure
Unable to automatically find API token with chromedp using URL http://jenkins.jx.35.234.62.96.nip.io/me/configure
Error: timeout waiting for initial target
Please go to http://jenkins.jx.35.234.62.96.nip.io/me/configure and click Show API Token to get your API Token
Then COPY the token and enter in into the form below:
```

Even I couldn't access the URL: http://jenkins.jx.35.234.62.96.nip.io/me/configure from browser giving not reachable.

### Steps to reproduce the behavior
**'jx create cluster gke'**
### Jx version

```
Using helmBinary helm with feature flag: none
NAME               VERSION
jx                 1.3.552
jenkins x platform 0.0.2940
Kubernetes cluster v1.9.7-gke.11
kubectl            v1.10.3
helm client        v2.11.0+g2e55dbe
helm server        v2.11.0+g2e55dbe
git                git version 2.19.0.windows.1
```

### Kubernetes cluster

What kind of Kubernetes cluster are you using & how did you create it?
Try to create gke cluster using the command  **'jx create cluster gke'**

### Operating system / Environment
OS:Windows10

### Expected behavior
It should suppose to successfully setup the jenkinsx in gke cluster
### Actual behavior
Getting error Unable to automatically find API token with chromedp using URL http://jenkins.jx.35.234.62.96.nip.io/me/configure.

@jstrachan - I try to follow your Youtube videos to setup the jenkinsx thru gke cluster, i am new to jenkinsx, please help me in resolving the error.

**Comment by okrasnyiblues** [comment]
I also get this error on eks

**Comment by jstrachan** [comment]
@okrasnyiblues sounds like ingress isn't working - what did you enter for the domain (did you let Jenkins X default to using your NLB's IP address with `.nip.io` appended?

**Comment by jstrachan** [comment]
@Nagamohan1 I wonder if this FAQ helps: https://jenkins-x.io/faq/issues/#jenkins-x-does-not-startup

it sounds like your cluster didn't start properly. Wonder if its not enough resources?

**Comment by okrasnyiblues** [comment]
I allowed Jenkins X to use default IP address that was created with it.
http://jenkins.jx.63.34.27.250.nip.io/me/configure
But i also found out that EIP that Jenkins X was using was not present in AWS console so I guess this might be the issue( there was EIP created by Jenkins X but it was not the same as one that Jenkins X was looking for).

**Comment by okrasnyiblues** [comment]
@jstrachan  Well, when i was saying EIP i meant elastic IP but apperently it's purpose is to change. If we speak about ELB, the IP in the ELB was same as IP Jenkins X was looking for. But still it couldn't manage to get token API.
So to make it more clear
Jenkins X was looking for http://jenkins.jx.63.34.27.250.nip.io/me/configure
IP in the ELB was 63.34.27.250
but Elastic IP was smth like 54.12.36.235. 

Right now I'm doing it one more time and will provide all the info if it doesn't work

**Comment by jstrachan** [comment]
@okrasnyiblues BTW ELB's don't get static IP addresses, they change every 24 hours - so we default to creating NLBs in AWS which have a fixed IP.

You shouldn't need to setup an EIP? Are you using the default ingress controller thats installed by Jenkins X?

**Comment by okrasnyiblues** [comment]
Yes, I now I get it that EIP isn't static IP. But yes I was using default ingress controller that was installed by Jenkins X

**Comment by jstrachan** [comment]
you using ELBs or NLBs? the latter has stable IP addresses, the former doesn't. How are you attaching the EIP?

**Comment by jstrachan** [comment]
I don't see the error - but also can't see your Jenkins either: http://jenkins.jx.34.247.230.220.nip.io/

**Comment by okrasnyiblues** [comment]
My bad, error wasn't included, it tried to get API Token few times, and I also wasn't able to connect to that nip.io and afterwards i deleted that cluster. 
Also I have an idea what might be the issue. I tried to create eks with region specified and everything work just fine:
`jx create cluster eks -n test -r eu-west-1 ` 
Note: in aws configure my region is set to eu-west-1, but jx default region is in us-west-2. When I created cluster without region it still was created in eu-west-1
